recipes/ndiff/build.sh

```diff
@@ -0,0 +1,11 @@
+mkdir -p $PREFIX/bin
+mkdir -p $PREFIX/include
+mkdir -p $PREFIX/lib
+mkdir -p $PREFIX/man/cat1
+mkdir -p $PREFIX/man/man1
+mkdir -p $PREFIX/share/lib/
```

Comment:
Please, delete these ``mkdir``

Reply:
Thanks for taking a look.  It looks like the "make install" command from the original package fails to create some of these directories if they do not exist ("bin" and "man/man1").   I can remove the others, but the conda build fails if I don't create "man/man1", since I guess the conda prefix will always have "bin".

Reply:
Remove the others which you don't need to create then :)